src/base/preferences.cpp

```diff
@@ -1611,10 +1642,9 @@ void Preferences::upgrade()
     QStringList labels = value("TransferListFilters/customLabels").toStringList();
     if (!labels.isEmpty()) {
         QVariantMap categories = value("BitTorrent/Session/Categories").toMap();
-        foreach (const QString &label, labels) {
+        foreach (const QString &label, labels)
```

Comment:
Yes - but my point is that this an automated change in the code cleanup commit - it's not part of the changes for the feature.